README.md

```diff
@@ -43,6 +55,31 @@ The plugin can also be configured to use multiple password files if required.
 </authConfigs>
 ```
 
+## Generating passwords using htpasswd
+You can use the [htpasswd](http://httpd.apache.org/docs/2.0/programs/htpasswd.html) program from Apache to manage your password file. You must use the -s option with htpasswd to force it to use SHA1 encoding for the password. So for example, you can use the following command to create a password file called "passwd" and put the password for the user "user" in it:
+
```

Comment:
`sha1` is considered insecure and not recommended, can we say so and a link to an issue indicating that other hashing mechanisms are WIP and contributions will be welcome?